CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+0.15.0 / 2021-04-13
+===================
+
+* **[Feature]**: Provide a way to inject an initial metric list for the autocompletion (#134)
+* **[Enhancement]**: Autocomplete metrics/function/aggregation when the editor is empty (#133)
+* **[Enhancement]**: Improve the documentation to reflect what the lib is providing. (#134)
+* **[Change]**: Export the essential interface in the root index of the lib. (#132)
+* **[Change]**: Downgrade the NodeJS version required (from 14 to 12) (#112)  
+* **[BreakingChange]**: Support CommonJS module. (#130) 
+  Be careful it changes the import path if you are using something not
```

Comment:
"Be careful it changes" -> I would change this to something like "Note that this release requires changes to your import paths if..."